R/droplet-actions.R

```diff
@@ -84,7 +84,7 @@
 #' }
 droplet_create <- function(name = random_name(),
                         size = getOption("do_size", "s-1vcpu-1gb"),
-                        image = getOption("do_image", "ubuntu-18-04-x64"),
+                        image = getOption("do_image", "rstudio-20-04"),
```

Comment:
What is `rstudio-20-04`? Is it just a blank ubuntu machine? Or have rstudio server installed? If the latter, I lean towards leaving it as a plain ubuntu machine - and perhaps creating a new function, e.g, `rstudio_create` if you want a similar functionality specifically for rstudio server. Thoughts?

Reply:
it's this image https://marketplace.digitalocean.com/apps/rstudio
it features R + Tidyverse

counterproposal, use "ubuntu-20-04" as default argument, and mention in the docs that "rstudio" shall give you a full setup (Even with RStudio Server) in 30 secs

Reply:
Either way -up to you now! 

But changing to `rstudio-20-04` does change the behavior of the function quite a bit in that it's not just a plain ubuntu machine anymore. I guess at least make sure the change is documented if you keep this change :)